manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```diff
@@ -28,6 +28,8 @@ spec:
 {{- end }}
     spec:
       serviceAccountName: secrets-store-csi-driver
+      affinity:
+{{ toYaml .Values.affinity | indent 8 }}
```

Comment:
We should have a separate affinity field for linux and windows. For example:

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/charts/secrets-store-csi-driver/values.yaml#L120-L121